package-lock.json

```diff
@@ -0,0 +1,1119 @@
+{
```

Comment:
Delete this `package-lock.json` file. In the future, only use `yarn` when interacting with the `regular-table` codebase. Never use `npm`